service/matching/task_queue_partition_manager.go

```diff
@@ -372,7 +376,7 @@ func (pm *taskQueuePartitionManagerImpl) unloadPhysicalQueue(unloadedDbq physica
 	delete(pm.versionedQueues, version)
 	pm.versionedQueuesLock.Unlock()
 	unloadedDbq.Stop()
-	pm.engine.updateTaskQueueGauge(pm, true, -1)
+	pm.engine.updatePhysicalTaskQueueGauge(pm, version, -1)
```

Comment:
please not that we do not want to emit the actual version value. we just want to know if it had a build id or a version set or non of them.